Dockerfile

```
@@ -83,22 +97,21 @@ FROM elixir:1.18-otp-27-slim AS runtime
 ENV LANG=C.UTF-8 \
     HOME=/app
 
-# Install runtime dependencies (minimal set)
-RUN apt-get update -y && \
+# Install runtime dependencies (minimal set) using BuildKit caching
+RUN --mount=type=cache,target=/var/lib/apt/lists \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)